### PR TITLE
Adding very rough instructions on Pi

### PR DESCRIPTION
**Pull request by Petermarcu**
Early instructions for folks who want to be on the bleeding edge of Arm32 and Raspberry PI support.

### CODE REVIEW

**Comment by Petermarcu** [comment]
@richlander , can you take a look. It's rough, early, and not intended to be perfect but I want to start pointing people to this to try things out.

**Comment by Petermarcu** [comment]
@blackdwarf, can you take a look?

File: samples/RaspberryPiInstructions.md [hunk 1]
```diff
@@ -0,0 +1,68 @@
+# .NET Core on Raspberry Pi
+
+Arm32 support for .NET Core is still being brought up but there has been enough progress for people to start playing with it. 
+It's not ready to be used as a developer environment yet but you can publish an application on a developer machine and run it on a Raspberry Pi. 
+These steps have been tested on both a Pi 2 and Pi 3 on Linux and Windows with success.
+
+##Creating an app:
+
+* [Install .NET Core](http://dotnet.github.io/getting-started/) into a supported developer configuration.
```
Comment (blackdwarf) [inline samples/RaspberryPiInstructions.md]: This link is non-existent anymore. You should point to dot.net/core.

File: samples/RaspberryPiInstructions.md [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+
+* From the terminal/commandline create a folder named helloworld and go into it.
```
Comment (blackdwarf) [inline samples/RaspberryPiInstructions.md]: "cd into it" would sound somewhat better than "go into it".

File: samples/RaspberryPiInstructions.md [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+* Under `./bin/Debug/netcoreapp1.1/publish` or `.\bin\Debug\netcoreapp1.1\publish` on Windows you need to edit the helloworld.runtimeconfig.json file to update the version to `1.2.0-beta-001206-00`.
+
+* This app is now ready to copy over to any machine that has the `1.2.0-beta-001206-00` version of .NET Core on it.
+
+
+##Getting the app to run on the Pi.
+
+###Linux (Ubuntu)
+
+* Install [Ubuntu 14.04 or 16.04](https://www.raspberrypi.org/downloads/) on your Pi.
```
Comment (blackdwarf) [inline samples/RaspberryPiInstructions.md]: Just as a curiosity, would Debian work as well?

Comment (Petermarcu) [inline samples/RaspberryPiInstructions.md]: Debian would work if we made the zips for them. I haven't tried these bits on debian but they may work. I don't know all the reasons why we have seperate debian and ubuntu builds.

**Comment by Petermarcu** [comment]
Thanks!

**Pull request merged**